packages/esm-patient-flags-app/src/flags/hooks/usePatientFlags.ts

```diff
@@ -12,23 +12,23 @@ interface FlagFetchResponse {
   auditInfo: { dateCreated: string };
 }
 
-interface FlagDefinition {
+export interface FlagDefinition {
   uuid: string;
   display: string;
   priority: { uuid: string; name: string };
   tags: Array<{ uuid: string; display: string }>;
 }
 
-interface FlagsFetchResponse {
+export interface FlagsFetchResponse {
   results: Array<FlagFetchResponse>;
 }
 
-interface FlagDefinitionsFetchResponse {
+export interface FlagDefinitionsFetchResponse {
   results: Array<FlagDefinition>;
 }
 
-interface FlagWithPriority extends FlagFetchResponse {
-  flagWithPriority?: FlagDefinition;
+export interface FlagWithPriority extends FlagFetchResponse {
+  flagDefinition: FlagDefinition;
```

Comment:
```suggestion
  flagDefinition?: FlagDefinition;
```

Should this be optional?

Reply:
I waffled on it but I think you're right, we should handle the case where, for whatever reason, the patient flag does not have a corresponding flag definition.